tests/scan/test_text_transformations.py

```diff
@@ -247,3 +247,11 @@ def test_typo_transformation():
     p = t.make_perturbation("If one doesn't know his mistakes, he won't want to correct them.")
 
     assert p == "If one doesn't know his misakes, he won't want to corrcet them."
+
+def test_ocr_typo_transformation():
+    from giskard.scanner.robustness.text_transformations import TextFromOCRTypoTransformation
+
+    t = TextFromOCRTypoTransformation(column="text", rng_seed=1)
+    p = t.make_perturbation("If one doesn't know his mistakes, he won't want to correct them.")
+
+    assert p == "1f 0ne d0ezn't kn0uu his nnistakes, he w0n't want to correct tbenn."
```

Comment:
The test doesn't pass for me:

```
Expected :"1f 0ne d0ezn't kn0uu his nnistakes, he w0n't want to correct tbenn."
Actual   :"If one doesn't know his mi5takes, he won't want to corrct them."

```

Reply:
Could you also add a test for the `len(x) < self.min_length` condition?

Reply:
I will check it and get it done.